src/test/java/teammates/test/cases/webapi/GetInstructorActionTest.java

```diff
@@ -23,13 +29,53 @@ protected String getRequestMethod() {
     @Test
     @Override
     protected void testExecute() throws Exception {
-        // TODO
+        InstructorAttributes instructor1OfCourse1 = typicalBundle.instructors.get("instructor1OfCourse1");
+        FeedbackSessionAttributes feedbackSessionAttributes = typicalBundle.feedbackSessions.get("session1InCourse1");
+
+        loginAsInstructor(instructor1OfCourse1.googleId);
+
+        ______TS("Not enough parameters");
+        verifyHttpParameterFailure();
+        verifyHttpParameterFailure();
+        verifyHttpParameterFailure(Const.ParamsNames.COURSE_ID, feedbackSessionAttributes.getCourseId());
+        verifyHttpParameterFailure(Const.ParamsNames.FEEDBACK_SESSION_NAME,
+                feedbackSessionAttributes.getFeedbackSessionName());
+        verifyHttpParameterFailure(Const.ParamsNames.COURSE_ID, feedbackSessionAttributes.getCourseId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, feedbackSessionAttributes.getFeedbackSessionName());
+
+        ______TS("typical success case");
+        String[] params = {
+                Const.ParamsNames.COURSE_ID, feedbackSessionAttributes.getCourseId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, feedbackSessionAttributes.getFeedbackSessionName(),
+                Const.ParamsNames.INTENT, Intent.INSTRUCTOR_SUBMISSION.toString(),
+        };
+
+        GetInstructorAction a = getAction(params);
+        JsonResult r = getJsonResult(a);
+
+        assertEquals(HttpStatus.SC_OK, r.getStatusCode());
+        InstructorData response = (InstructorData) r.getOutput();
+        assertEquals(instructor1OfCourse1.name, response.getName());
     }
 
     @Test
     @Override
     protected void testAccessControl() throws Exception {
-        // TODO
+        InstructorAttributes instructor1OfCourse1 = typicalBundle.instructors.get("instructor1OfCourse1");
+        FeedbackSessionAttributes fs = typicalBundle.feedbackSessions.get("session1InCourse1");
+
+        loginAsInstructor(instructor1OfCourse1.googleId);
+
+        ______TS("only instructors of the same course can access");
```

Comment:
This is a positive test case - it asserts that _____ can happen. We also need to cover the negative cases (_____ _cannot_ happen) such as asserting that `instructors of other courses cannot access`. Otherwise, it can be quite difficult to catch regressions such as if someone introduced a bug that causes such a security breach.

This might happen to other tests too. Take a look through them and see if you need to beef them up.

Reply:
Yes correct! I have looked through most of it and beefed up the tests.